nextflow/workflows/copyFiles.nf

```diff
@@ -0,0 +1,105 @@
+import java.text.SimpleDateFormat
+
+src_ch = Channel.fromPath("${params.source}/*${params.file_format}")
+dest_ch = Channel.value(params.destination)
+
+log.info """\
+        R S Y N C - N F   P I P E L I N E
+        ============================================================
+        source     : ${params.source}
+        destination: ${params.destination}
+        """
+        .stripIndent(true)
+
+// Helper function to check if a directory exists
+def checkIfExists(path, label) {
+    if (!file(path).exists()) {
+        error "${label} directory doesn't exist: ${path}"
+    }
+}
+
+// Function to build and send email
+def sendPipelineStatusEmail(String pipelineName, String status, String recipient) {
```

Comment:
Could we simplify this by using the built-in Nextflow workflow notification report with `-N`? e.g. https://www.nextflow.io/docs/latest/notifications.html#workflow-notification

Reply:
The reason why I though of creating a custom email is because `-N` doesn't mention the source and destination locations.

But if it's not needed, we can use `-N` feature as you suggested